lib/ioc-rc

```diff
@@ -555,7 +557,8 @@ __rc_jails () {
         local boot_order=$(sort -n $boot_list)
         local shutdown_order=$(sort -rn $boot_list)
     else
-        __info "no jails found with property boot=on, exiting.."
+        __info "no jails found with property boot=on or
+        with jail_hostid: $hostid on this host, exiting.."
```

Comment:
Nope just line 561. You could also move it over to match up with the " in `__info "no jails found with property boot=on or`